abel/basex.py

```diff
@@ -402,33 +421,19 @@ def _bs_basex(n_vert=1001, n_horz=501,
             # End of vectorized third loop
             aux *= 2
 
-            M_horz[Rm_h - 1 + l, k] = aux  # All rows below center
-            M_horz[Rm_h - 1 - l, k] = aux  # All rows above center
+            M[Rm_h - 1 + l, k] = aux  # All rows below center
+            # M[Rm_h - 1 - l, k] = aux  # All rows above center
```

Comment:
Which?

This method must be refactored, but I did not understand yet how it works. Partially, because there is no human-readable description. If these are removed, it will become even more cryptic.

Reply:
I meant that in
```py
            M[Rm_h - 1 + l, k] = aux  # All rows below center
            # M[Rm_h - 1 - l, k] = aux  # All rows above center
```
the second line is a copy paste of the fist, but commented out, and could be removed..

Reply:
It is not. Notice the sign difference:
<pre>
           ↓
M[Rm_h - 1 <b>+ l</b>, k] = aux  # All rows <b>below</b> center
M[Rm_h - 1 <b>- l</b>, k] = aux  # All rows <b>above</b> center
</pre>